sw/device/tests/sim_dv/pwrmgr_random_sleep_all_reset_reqs_test.c

```diff
@@ -503,6 +503,8 @@ bool test_main(void) {
             rst_info == kDifRstmgrResetInfoSw,
         "Wrong reset reason %02X", rst_info);
 
+  LOG_INFO("New reset event");
```

Comment:
This test runs one of two sleep mode chosen by random.
Can you run with a few different seed to make sure this patch works for both sleep mode?

Reply:
Yes of course.
So far I've tested it with 10 different seeds and all of them pass.
I plan to run another 20 just to be sure.

Reply:
that sounds great!